src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -128,7 +131,7 @@ namespace winrt::TerminalApp::implementation
         void _CloseWarningPrimaryButtonOnClick(Windows::UI::Xaml::Controls::ContentDialog sender, Windows::UI::Xaml::Controls::ContentDialogButtonClickEventArgs eventArgs);
         void _ThirdPartyNoticesOnClick(const IInspectable& sender, const Windows::UI::Xaml::RoutedEventArgs& eventArgs);
 
-        void _HookupKeyBindings(TerminalApp::AppKeyBindings bindings) noexcept;
+        void _HookupKeyBindings(TerminalApp::KeyMapping keymap) noexcept;
```

Comment:
nit: may want to change signature to `const KeyMapping&` just to save the refcount